web/src/services/localization.ts

```diff
@@ -67,5 +69,6 @@ export async function createBundleGenerator(
     ])
   );
 
-  return asBundleGenerator(localeMessages, messageOverwrites);
+  return CachedSyncIterable.from(
+    asBundleGenerator(localeMessages, messageOverwrites));
```

Comment:
I had to pull in the `cached-iterable` dependency to ensure that the generator is not depleted on re-renders of the `LocalizationProvider`. I'm working on streamlining the API to make it easier to use and not require such obscure measures.

Reply:
I published `@fluent/react` 0.12 which makes the API of `<LocalizationProvider>` simpler. It doesn't take the bundles generator as a prop anymore. Instead, the bundles generator can be used to instantiate a `ReactLocalization` which can then be passed to `<LocalizationProvider>` as well as stored in the state safely.